Alignment/CommonAlignmentProducer/test/GlobalPositionRcdRead.cpp

```diff
@@ -14,8 +14,8 @@
 class GlobalPositionRcdRead : public edm::EDAnalyzer {
 public:
   explicit GlobalPositionRcdRead(const edm::ParameterSet& iConfig) : nEventCalls_(0) {}
-  ~GlobalPositionRcdRead() {}
-  virtual void analyze(const edm::Event& evt, const edm::EventSetup& evtSetup);
+  ~GlobalPositionRcdRead() override {}
```

Comment:
These changes look like what an old code-checks would do.
is this PR mixing expansion of checks to the /test directory with the new type of checks?
It may be better to separate the broader scope of what we were doing already from the new checks

Reply:
we do not run clang-tidy for files under `test` directory. @mrodozov  please fix the PR and do not run clang-tidy for test files.

Reply:
There is https://github.com/cms-sw/cmssw/pull/30508 now without the /test/ dirs